packages/react/package.json

```
@@ -44,5 +44,8 @@
   "devDependencies": {
     "@types/react": "^18.2.79",
     "react": "^18.2.0"
+  },
+  "peerDependencies": {
+    "@synergy-design-system/tokens": "workspace:*"
```

should be `workspace:^` version

```suggestion
    "@synergy-design-system/tokens": "workspace:^"
```

---

Done :)